src/org/ods/OdsContext.groovy

```diff
@@ -45,7 +45,7 @@ class OdsContext implements Context {
     config.nexusPassword = script.env.NEXUS_PASSWORD
     config.openshiftHost = script.env.OPENSHIFT_API_URL
     config.bitbucketHost = script.env.BITBUCKET_HOST
-    config.odsSharedLibVersion = script.sh(script: "env | grep 'library.ods-library.version' | cut -d= -f2", returnStdout: true, label : 'getting ODS shared lib version')
+    config.odsSharedLibVersion = script.sh(script: "env | grep 'library.ods-library.version' | cut -d= -f2 | tr -d '\\n\\r'", returnStdout: true, label : 'getting ODS shared lib version')
```

Comment:
wouldn't it be easier to use `.trim()` at the end?

Reply:
Yes, this would be easier, even with my brain whispering, "but there are piped shell commands..." :-D